plugins/modules/mysql_query.py

```diff
@@ -139,6 +148,18 @@
 # Module execution.
 #
 
+
+def execute_and_return_time(cursor, query, args):
+    # Measure query execution time in milliseconds
+    start_time = time.perf_counter()
+
+    cursor.execute(query, args)
+
+    # Calculate the execution time rounding it to 4 decimal places
+    exec_time_ms = round((time.perf_counter() - start_time) * 1000, 4)
```

Comment:
Why *1000? Isn't time.perf_counter() already in ms?

Reply:
i found that it returns time in seconds:
```
The perf_counter() function always returns the float value of time in seconds.
```

Reply:
in case of one query it returns
```
    "execution_time_ms": [
        6.248
    ],
```

in case of 3 queries:
```
    "execution_time_ms": [
        0.2793,
        0.1921,
        0.1112
    ],
```
Yeah it's floating so it's kinda tricky to tests against any hardcoded values

Reply:
copied from local test run ^

Reply:
I just get it. Line 153 is the doc for the method, not just line 154! So it make sense now. You can ignore my question. Thanks for the precision!

Reply:
@laurent-indermuehle sure, if it's ok now, feel free to merge